### ISSUE TITLE
[BUG] (Semi-resolved) Steam CMD Times out when trying to download workshop mods

### ISSUE THREAD

**Issue by RudyRedTop** [Foxlider/FASTER#55]
I just recently got back into server hosting after not using FASTER for upwards of 4-5 months. I imported a new mod HTML with the new thing we want to use. But every time I go to have it download all the new mods it get's through 1-4 mods before it throws this error and times out. 

**To Reproduce**
1. I simply click on the update-all button on the steam mods section and let it run. 
2. Eventually it will time out and close Steam CMD.

![AnyDesk_4zggSw74qc](https://user-images.githubusercontent.com/96358139/146658089-ad0a2d46-e870-4c84-afc5-46ea852c3fb9.png)

![AnyDesk_VptQPeMt23](https://user-images.githubusercontent.com/96358139/146658134-eecd2207-ff97-4ca0-8aa0-807a89e38315.png)

Literally as I ran it again just to get a third screenshot, it miraculously decided to work and download all of the other mods on the list. So here are the screenshots showing it work I guess.

![AnyDesk_ML3rtMDTHA](https://user-images.githubusercontent.com/96358139/146658666-bb3eeac6-0711-4d67-bb48-3e344a2c8578.png)
![AnyDesk_WPAda6Wdqb](https://user-images.githubusercontent.com/96358139/146658668-6b56deee-d58f-43e2-9414-7c43d663408b.png)

 - OS: Windows 10 Home build 19042
 - Faster Version 1.7.3.0

**Comment by Foxlider** [comment]
It's a known bug.

TLDR : 
Reason : SteamCMD sucks
Solution : Keep trying. Or wait a few days.

Long version : 
SteamCMD dies when it it tries to download more than 1Gb.
It keeps the downloaded mod in cache so keep trying till it works. 
FASTER is sadly unable to detect that it fails for that particular reason so I couldn't make a system that just retries without risking to make an endless loop of retries if SteamCMD somehow just got stuck.
Thankfully I got a new system working with the release of 1.8. I'm almost done ironing out bugs and should by out by Christmas

**Comment by RudyRedTop** [comment]
> 

Gotcha, thanks for all the hardwork you put into this man. I look forward to 1.8!

**Comment by Foxlider** [comment]
Fixed with 1.8
